src/rattler_build_conda_compat/jinja.py

```diff
@@ -36,6 +57,26 @@ def jinja_env() -> jinja2.Environment:
         undefined=_MissingUndefined,
     )
 
+    # inject rattler-build recipe functions in jinja environment
+    env.globals.update(
+        {
+            "compiler": lambda x: x + "_compiler_stub",
+            "stdlib": lambda x: x + "_stdlib_stub",
+            "pin_subpackage": stub_subpackage_pin,
+            "pin_compatible": stub_compatible_pin,
+            "cdt": lambda *args, **kwargs: "cdt_stub",  # noqa: ARG005
+        }
+    )
+
+    # inject rattler-build recipe filters in jinja environment
+    env.filters.update(
+        {
+            "version_to_buildstring": version_to_build_string,
+            "split": split_filter,
+        }
+    )
+    return env
```

Comment:
Do we also need to add some stub functions for `env.get` and `env.get_default`?

Reply:
All the more reason tho have an actual recipe that contains some of these corner cases.

Reply:
that's true , added mamba_recipe as a test one